parfive/downloader.py

```diff
@@ -72,6 +74,17 @@ def __init__(self, max_conn=5, progress=True, file_progress=True,
 
         self.overwrite = overwrite
 
+        self.proxy = proxy
+        self.proxy_auth  = None
+
+        self.ua_header = {
```

Comment:
This should be done in `_get_http` so that we give the user a chance over overwrite it.